libmscore/figuredbass.cpp

```diff
@@ -1247,7 +1247,7 @@ void FiguredBass::draw(QPainter* painter) const
       if (!score()->printing() && score()->showUnprintable()) {
             for (qreal len : _lineLengths) {
                   if (len > 0) {
-                        painter->setPen(QPen(Qt::lightGray, 1));
+                        painter->setPen(QPen(Qt::lightGray, 3));
```

Comment:
My guess is, this should be scaled according to screen resolution somehow.

Reply:
I can trust you, as I have no guess. The new value gives me on ver. 3.x more or less the same result as the old value gives in ver. 2.x (same machine, same OS, same screen). As the 2.x result received no complains — as far as I can remember — I expect the new value can be acceptable.

Also, no other size I can see around there seems to be scaled according to screen resolution; I would assume it is already taken into account in setting Qt drawing space up but, as I said, I have no guess in this matter...